rdtools/analysis_chains.py

```diff
@@ -651,8 +732,12 @@ def _clearsky_preprocess(self):
         self._filter(cs_normalized, 'clearsky')
         cs_aggregated, cs_aggregated_insolation = self._aggregate(
             cs_normalized[self.clearsky_filter], cs_insolation[self.clearsky_filter])
-        self.clearsky_aggregated_performance = cs_aggregated
-        self.clearsky_aggregated_insolation = cs_aggregated_insolation
+        # Run daily filters on aggregated data
+        self._aggregated_filter(cs_aggregated, 'clearsky')
+        # Apply daily filter to aggregated data and store
+        self.clearsky_aggregated_performance = cs_aggregated[self.clearsky_filter_aggregated]
+        self.clearsky_aggregated_insolation = \
+            cs_aggregated_insolation[self.clearsky_filter_aggregated]
 
```

Comment:
@kperrynrel  - I'm still getting errors when I exercise the aggregated filter on clearsky conditions.  On lines 703-706 of `_sensor_preprocess` you re-index the data, but don't do the same thing here in `_clearsky_preprocess`.  Copying those four lines into here to re-establish the originally indexed time series will fix the errors.  Thanks!